File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for c in commands:
        x = c[0]
        y = c[1]
        z = c[2]
        
        li = []
        for j in range(x-1, y):
            li.append(array[j])
            
        li.sort()
        answer.append(li[z-1])
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int idx = 0;
        for(int[] command: commands) {
            int i = command[0];
            int j = command[1];
            int k = command[2];
            int[] temp = Arrays.copyOfRange(array, i - 1, j);
            Arrays.sort(temp);
            answer[idx] = temp[k - 1];
            idx++;
        }
        return answer;
    }
}